### ISSUE TITLE
Do away with "Optional Metadata" on submission forms

### ISSUE THREAD

**Issue by monika-aashe** [AASHE/hub#73]
"Optional Metadata" is problematic for a few reasons:
- Author, image, file, website are standard - Not all of these are relevant for every content type
- In many cases, the fields are required, so there is nothing optional about them. 
- UI - users may miss the other tabs

Decision:
- [x] Remove tabs - instead, separate "sections" with horizontal lines or something
- [ ] Only ask for author, image, file, website if it is needed, 
- [ ] Need to pay attention to whether fields are optional or required - not standard for all content types

See revised summary tab for more info (rows 23-26): https://docs.google.com/spreadsheets/d/1ADsnCZXKU1lhA4GHc2xhsY5ddLRi7kgvYJLSTHZKlmw/edit#gid=935008516

**Comment by monika-aashe** [comment]
Also, we need to only ask the "affirmation of ownership" once on the form, not for every file submitted. Ideally this will be conditionally required (only required if a file is uploaded).

**Comment by monika-aashe** [comment]
I do really like the way the metadata looks in the approved resources design-wise, and how the author info and files are separated and use icons. Would be great to keep this somehow.

**Comment by monika-aashe** [comment]
Moved some content into a separate issue - #120

**Comment by jamstooks** [comment]
Closed #120, in the form-field-ordering branch (12765b054e701a637a357a1c51e28cb03d370f77), but still need the following for reference:

> 9. Unique fields per content type (only some content types)
>    * -------- break (horizontal line)
> 10. Website (include everywhere but Conf. Pres., Photographs)
> 11. Image/document upload (only some content types)
> 12. Affirmation of ownership (only some content types)
>    * -------- break (horizontal line)

**Comment by jamstooks** [comment]
The majority of this ticket has been addressed, but there are still some outliers because this ticket covers so much. I've updated the description of the ticket to include a ticket to indicate status.

I'd like some feedback from @monika-aashe and @juliandsmith on the functionality for author/files/images regarding the "add" buttons. Keep in mind that the other pieces of this ticket are yet to be addressed. Based on other tickets, I might do the same for websites.
